src/reducers/index.js

```diff
@@ -67,17 +68,19 @@ export const populatePropertyDefaults = (propertyTemplate) => {
 }
 
 export const refreshResourceTemplate = (state, action) => {
-  const newState = { ...state }
+  const rtId = Object.keys(state.resource).pop()
```

Comment:
This variable expects that there is an existing resource state, so I removed the test that (AFAICT) artificially creates a missing payload.

Reply:
rtId is not a very expressive name.  How about `resourceTemplateId` instead?